README.md

```diff
@@ -20,13 +20,13 @@ Package and installation instructions are available at the [Maven Central Reposi
 <dependency>
   <groupId>eu.kevin</groupId>
   <artifactId>kevin-jvm</artifactId>
-  <version>0.3.0</version>
+  <version>1.0.0</version>
 </dependency>
 ```
 
 ### Gradle
 ```
-implementation 'eu.kevin:kevin-jvm:0.3.0'
+implementation 'eu.kevin:kevin-jvm:1.0.0'
```

Comment:
Why version 1?

Reply:
I believe it's breaking. If someone is already using Ktor 1.6.x in their project or Ktor 1.6.x is coming from another dependency, it's compatibility issue